### DISCUSSION TITLE
Web demo cannot connect device: Error: Failed to execute 'claimInterface' on 'USBDevice': Unable to claim interface.

### DISCUSSION THREAD

**Discussion by tiennguyen12g** [yume-chan/ya-webadb#520] [Q&A] [answered]
Hello everyone, i get this error:
"Error: Failed to execute 'claimInterface' on 'USBDevice': Unable to claim interface."
If you have experience about this, can you help me? thank all you

**Comment by yume-chan** [comment, marked as answer]
Tango can't work along with Google ADB, do you have ADB or other Android management tool running?

**Reply by tiennguyen12g** [reply]
I run adb on system, do i need to shutdown it before using Demo. Thank you.

**Reply by tiennguyen12g** [reply]
Thank bro, it work after "adb kill-server".
